herddb-core/src/main/java/herddb/core/TableSpaceManager.java

```diff
@@ -514,6 +524,43 @@ void apply(CommitLogResult position, LogEntry entry, boolean recovery) throws Da
                 writeTablesOnDataStorageManager(position, false);
             }
             break;
+            case LogEntryType.TABLE_CONSISTENCY_CHECK: {
+                try {
+                    TableChecksum check = new ObjectMapper().readValue(entry.value.to_array(), TableChecksum.class);
```

Comment:
this line returns null when the follower executes it

Reply:
Is it entry.value null? Or the Renault of readValue?
You can add more log

Reply:
I added this debug:
```
if(!isLeader()){
	System.out.println("Follower entry value = "  + entry.value);
}else{
	System.out.println("Leader entry value = "  + entry.value);
}
```
For the leader i have:
`Leader entry value = 7b227461626c6553706163654e616d65223a227432222c227461626c654e616d65223a227461626c6531222c22646967657374223a2d313230353033343831393633323137343639352c2264696765737454797065223a2253747265616d696e675858486173683634222c226e756d5265636f726473223a322c226e6578744175746f496e6372656d656e7456616c7565223a312c227175657279223a2253454c45435420206b65792046524f4d207461626c6531206f72646572206279206b6579222c227363616e4475726174696f6e223a317d`

But for follower node i have:
`Follower entry value = null`

Reply:
I solved the problem. I forgot to write the values ​​in the entry :( ...
But I have another question:
When I run the select query in the follower, he complains that he is not the leader of the tablespace. Is it possible to unblock the follower to run the query in his local copy of his data?

Reply:
Yes you can.
Maybe you have to add a field to Statement like 'allowExecutionFromFollower'

Reply:
Use only one static ObjectMapper